Sources/Commands/SwiftTool.swift

```diff
@@ -591,61 +599,12 @@ public class SwiftTool<Options: ToolOptions> {
         }
         try createSymlink(oldBuildPath, pointingAt: plan.buildParameters.buildPath, relative: true)
     }
-    
+
     func runLLBuild(manifest: AbsolutePath, llbuildTarget: String) throws {
         assert(localFileSystem.isFile(manifest), "llbuild manifest not present: \(manifest.asString)")
-        
-        // Create a temporary directory for the build process.
-        let tempDirName = "org.swift.swiftpm.\(NSUserName())"
```

Comment:
We shouldn't remove this just yet. I thought we were going to land this with flag and switch to the default once we have confirmed it with OSS projects.

Reply:
Sorry, that was a mistake!